packages/components/src/Text/Text.spec.tsx

```diff
@@ -23,4 +23,24 @@ describe("<Text />", () => {
       </p>
     `);
   });
+
+  it("should pass the passthrough ref", async () => {
```

Comment:
I'm a big fan of tests. Wondering if it's more trouble than it's worth in this case, though?

Reply:
I mean, I can remove it, but I didn't like if this were somehow to break, we could be causing accessibility issues where we're trying to focus on a modal or whatever and no one notices. Is there any reason to not have it when it's already here?

Reply:
Nope, no reason to remove it since it's already there.

To me, it's not always valuable things we technically should, but are mostly about wiring things up (such as passing a ref through forward ref to the component). But that's my 1.5 cents, and this is a reasonable test 👍. Especially since you find value in it.